utils/fix_cmocka_xml.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+'''
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+
+'''
+import os
+
+# Some fixes for the CMOCKA headers
+xml_header ="<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n"
+parent_header = "<testsuites>\n"
+parent_footer = "</testsuites>\n"
+file_extensions = "*.xml"
+path = os.path.join(os.getcwd(), "..", "test_results") + os.path.sep
```

Comment:
As commented before, you don't want `..` in the path.  That was only relevant when you were building a path from `__file__`

Reply:
.. is needed... Otherwise, path will point to the location where files are not available.

Reply:
The test result seems to say otherwise:

```
[2019-05-07T21:22:33.205Z] OSError: [Errno 2] No such file or directory: '/data/jenkins/jenkins-1/ci_vm_wolf-105vm10_12/workspace/daos-stack_daos_PR-425/../test_results/'
```

The `..` in the above path is actually looking outside of the test run's own namespace (`/data/jenkins/jenkins-1/ci_vm_wolf-105vm10_12/workspace/daos-stack_daos_PR-425`).

Reply:
Oh. Now, I get it...
In Jenkinsfile, we do this.
cd $DAOS_BASE
utils/run_test.sh
... Python script is also invoked utils/fix_cmocka_xml.py... Interesting...